components/profile/profile.css

```diff
@@ -1,12 +1,16 @@
+.main-profile {
+  max-width: 1200px;
+  margin: auto;
+}
+
 .profile {
   width: 90%;
   margin: auto;
-  margin-bottom: 30px;
-  display: flex;
-  flex-direction: row-reverse;
-  justify-content: space-between;
+  margin-bottom: 50px;
+  text-align: center;
   color: #fff;
```

Comment:
you could put it in the :root as well, just in case you want to change it later on